packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -17,7 +17,7 @@ contract L1StandardBridge_Getter_Test is Bridge_Initializer {
         assert(L1Bridge.OTHER_BRIDGE() == L2Bridge);
         assert(L1Bridge.messenger() == L1Messenger);
         assert(L1Bridge.MESSENGER() == L1Messenger);
-        assertEq(L1Bridge.version(), "1.1.0");
+        assertEq(L1Bridge.version(), "1.1.1");
```

Comment:
we could prob delete this assertion, its kind of overhead

Reply:
Split out into #6119